migrations/20200422002041_add_messaging_sid_to_campaign.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function(knex) {
+  return knex.schema.table("campaign", table => {
+    table.string("messaging_service_sid").defaultTo(null);
```

Comment:
Can we rename this to `messageservice_sid` to match the `message.messageservice_sid` column?